apps/web/components/templates/challenges/all/allChallenges.tsx

```diff
@@ -0,0 +1,44 @@
+import { ChallengeItem } from 'molecules/challengeItem/challengeItem';
+
+type challenge = {
+	title: string;
+	description: string;
+	designs: string;
+	difficulty: string;
+	image: string;
+};
+
+const challenges: challenge[] = [
+	{
+		title: 'First Challenge',
+		description:
+			'This is example of challenge. In hac habitasse platea dictumst. Praesent eu auctor velit. Cras mattis gravida odio euismod placerat. Morbi vestibulum dapibus diam, a lacinia felis porttitor vel. Ut sodales tincidunt orci, vel condimentum nibh. Fusce in elit euismod, laoreet felis vel, rutrum lacus.',
+		designs: '',
+		difficulty: 'medium',
+		image:
+			'https://images.unsplash.com/photo-1498008122250-bcb854c8462d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2070&q=80',
+	},
+	{
+		title: 'Second Challenge',
+		description:
+			'This is example of challenge. In hac habitasse platea dictumst. Praesent eu auctor velit. Cras mattis gravida odio euismod placerat. Morbi vestibulum dapibus diam, a lacinia felis porttitor vel. Ut sodales tincidunt orci, vel condimentum nibh. Fusce in elit euismod, laoreet felis vel, rutrum lacus.',
+
+		designs: '',
+		difficulty: 'advanced',
```

Comment:
I don't see if `designs` property is used anywhere in this PR, could you say what's the purpose of it? ☺️

Reply:
`designs` is used in `newChallengeForm`, i don't created appearance for this in list of all challenges yet.